lib/logstash/outputs/gcs/client.rb

```diff
@@ -0,0 +1,71 @@
+# Author: Joseph Lewis III <jlewisiii@google.com>
+# Date: 2018-02-23
+#
+# Copyright 2018 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
```

Comment:
we don't include such headers anywhere in our code - I am guessing this is an editor default for new files - would you mind removing it?

Reply:
Sure, the main ruby file has the same thing there which I can remove if you like as well. I just have to add Google LLC to the contributors list.